File: Python projects/Python Fundamentals/functions/functions_exercise/02_add_and_subtract.py
def sum_numbers(num_1, num_2):
    return num_1 + num_2


def subtract(num_1, num_2):
    return num_1 - num_2


def add_and_subtract(num_1, num_2, num_3):
    return subtract(sum_numbers(num_1, num_2), num_3)


print(add_and_subtract(int(input()), int(input()), int(input())))


'''
TASK:
You will receive three integer numbers. 
Write functions named:
sum_numbers() which returns the sum of the first two integers
subtract() which returns the difference between the returned result of the first function and the third integer. 
Wrap the two functions in a function named add_and_subtract() which will receive the three numbers as parameters.
Print the result of the subtract() function on the console.
'''

File: JavaScript projects/JavaScript Fundamentals/04.Functions/02.Exercise/02_add_and_subtract.js
function sumAndSubtract(first, second, third){

    let sum = (x, y) => x + y;
    let sub = (x, y) => x - y;

    let sumResult = sum(first, second);
    return sub(sumResult, third); 
}

console.log(sumAndSubtract(23, 6, 10));
console.log(sumAndSubtract(1, 17, 30));
console.log(sumAndSubtract(42, 58, 100));
